src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -14,20 +15,21 @@ public static long countWords() throws IOException {
         String contents = new String(Files.readAllBytes(Paths
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
-
-        long count = 0;
-        for (String w : words) {
-            if (w.length() > 12) count++;
-        }
-        return count;
+        return words.stream().filter(word -> word.length() > 12).count();
     }
 
     public static void printLongestWordTop100() throws IOException {
         String contents = new String(Files.readAllBytes(Paths
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        words.stream()
+                .filter(word -> word.length() > 12)
+                .sorted(Comparator.comparing(String::length))
+                .distinct()
```

Comment:
```suggestion
                .distinct()
                .sorted(Comparator.comparingInt(String::length).reversed())
```
- 길이가 가장 긴 문자열이니 역순이어야 하지 않을까요?
- 지금은 별 차이 없지만, 데이터가 많을 경우 distinct() 후에 sorted()하는 것과 성능 차이가 있을 수 있어요.

Reply:
테스트 코드를 잘못 작성했군요! ㅜㅜ
distinct()와 sorted() 순서도 주의해야겠군요.
꼼꼼한 리뷰 감사합니다.